src/modules/DynamicAssessment/PageAssessment.tsx

```diff
@@ -26,7 +26,7 @@ export default function PageAssessment({
   filteredOptions: LiveOption[];
   children: ReactNode;
 }) {
-  const [isComplete, setIsComplete] = useState(false);
+  const [isComplete, setIsComplete] = useState(true);
```

Comment:
Was this for testing? :laughing:

Reply:
😆 yep, just fixed it

Reply:
There's also this line:
`const score = 4 || getAssessmentScore(questions);`